cards/common-partials/thumbsfeedback.hbs

```diff
@@ -0,0 +1,29 @@
+<form class="HitchhikerCard-thumbs js-HitchhikerCard-feedbackForm">
```

Comment:
We could because it's almost exactly the same. The only difference is that direct answers card feedback has text that accompanies the buttons. If we wanted to combine them, it wouldn't be hard

Reply:
If we can combine them, that would be great, but I would prioritize the positioning of the buttons first